_blog/2018/soft-multitenancy.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Istio Soft Multitenancy Support"
+overview: Using Kubernetes namespace and RBAC to create an Istio soft multitenancy environment
+publish_date: March 21, 2018
+subtitle: Using multiple Istio control planes and RBAC to create multitenancy
+attribution: John Joyce and Rich Curran
+
+order: 92
+
+layout: blog
+type: markdown
+redirect_from: "/blog/soft-multitenancy.html"
+---
+{% include home.html %}
+Multitenancy is commonly used in many environments across many different applications,
+but the implementation details and functionality provided on a per tenant basis does not
+follow one model in all environments.  The [Kubernetes multitenancy working group](
+https://github.com/kubernetes/community/blob/master/wg-multitenancy/README.md)
+is working to define the multitenant use cases and functionality that should be available
+within Kubernetes. However, from their work so far it is clear that only "soft multitenancy"
+is possible due to the inability to fully protect against malicious containers or workloads
+gaining access to other tenant's pods or kernel resources.
+
+## Soft multitenancy
+For this blog, "soft multitenancy" is defined as having a single Kubernetes control plane
+with multiple Istio control planes and multiple meshes, one control plane and one mesh
+per tenant. The cluster administrator gets control and visibility across all the Istio
+control planes, while the tenant administrator only gets control of a specific Istio
+instance. Separation between the tenants is provided by Kubernetes namespaces and RBAC.
+
+One use case for this deployment model is a shared corporate infrastructure where malicious
+actions are not expected, but a clean separation of the tenants is still required.
+
+Potential future Istio multitenant deployment models are described at the bottom of this
+blog.
+
+**Note:** This blog is a high-level description of how to deploy Istio in a
```

Comment:
Instead of **Note**, use > at the start of the line to create a block quote.

Reply:
Done